autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -516,6 +670,31 @@ def file_output(
             Y_valid_pred: np.ndarray,
             Y_test_pred: np.ndarray
     ) -> Tuple[Optional[float], Dict]:
+        """
+        This method decides what file outputs are written to disk.
+
+        It is also the interface to the backed save_numrun_to_dir
+        which stores all the pipeline related information to a single
+        directory for easy identification of the current run.
+
+        Args:
+            Y_optimization_pred (np.ndarray):
```

Comment:
Uum, can we change the name like `Y_cv_pred`?
It is the performance on the cross validation (or holdout validation) rather than optimized performance, right? (Another reason why I thought it is strange is that we would not know in terms of what we optimize the performance.)

Reply:
I would like to honor the title of the PR and only address documentation-related changes. This looks like a refactor.